modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -339,8 +339,16 @@ const questionnaireStyle = theme => ({
         float: "right",
         margin: theme.spacing(1),
     },
-    titleButton: {
-        float: "right"
+    actionsMenu: {
+        border: "1px solid #ddd",
+        borderRadius: theme.spacing(3),
+        display: "flex",
+        float: "right",
+        marginRight : theme.spacing(2),
+        marginTop: theme.spacing(1)
+    },
+    actionsMenuItem: {
+        padding: theme.spacing(0,1,0,1)
```

Comment:
Let's make this look more like a menu"
```suggestion
        padding: theme.spacing(0,1),
        "& .MuiButtonBase-root" : {
          fontWeight: "normal",
          justifyContent: "flex-start",
          textTransform: "none",
          width: "100%",
       }

```